doc/classes/OS.xml

```diff
@@ -578,12 +628,14 @@
 				OS.MoveToTrash(ProjectSettings.GlobalizePath(fileToRemove));
 				[/csharp]
 				[/codeblocks]
+				[b]Note:[/b] This method is implemented on Android, Linux, macOS, and Windows.
```

Comment:
Actually I realize that the other similar statements don't use the Oxford comma in this file. Maybe it's best to remove it for consistency.

Then a unification pass could be done in another PR as we currently use a mix of both:
```
$ rg -g'*.xml' "is implemented on .*, and [A-Za-z]+\." | wc -l
18
$ rg -g'*.xml' "is implemented on .*[^,] and [A-Za-z]+\." | wc -l
39
```

18 with Oxford comma, 39 without.

Reply:
Okiedokie then, unless we want to explicitly make an exception when listing platform availability.